C7Engine/EntryPoints/TurnHandling.cs

```diff
@@ -69,6 +69,8 @@ internal static void AdvanceTurn()
 						newUnit.location = tile;
```

Comment:
just a note for later, this whole section is a prime candidate for refactoring into some factory methods as mentioned in #238, maybe something like GetNextRandomUnit(rng) which calls UnitPrototype.GetInstance()